src/network/netdev/netdev-gperf.gperf

```diff
@@ -79,6 +80,15 @@ VXLAN.MaximumFDBEntries,     config_parse_unsigned,                0,
 VXLAN.PortRange,             config_parse_port_range,              0,                             0
 VXLAN.DestinationPort,       config_parse_destination_port,        0,                             offsetof(VxLan, dest_port)
 VXLAN.FlowLabel,             config_parse_flow_label,              0,                             0
+GENEVE.Id,                   config_parse_uint32,                  0,                             offsetof(Geneve, id)
+GENEVE.Remote,               config_parse_geneve_address,          0,                             offsetof(Geneve, remote)
+GENEVE.TOS,                  config_parse_unsigned,                0,                             offsetof(Geneve, tos)
+GENEVE.TTL,                  config_parse_unsigned,                0,                             offsetof(Geneve, ttl)
```

Comment:
the man page you added says the range for this is 1-155 (is that true, isn't that a typo? should that be 255?). If so, then please add config_parse_geneve_ttl() or so, which ensures the value passed is in the right range.